packages/react/src/Box/Box.tsx

```diff
@@ -0,0 +1,201 @@
+import {default as clsx} from 'clsx'
+import React, {PropsWithChildren, useMemo} from 'react'
+import {AnimateProps, useAnimation} from '../animation'
+import type {BaseProps} from '../component-helpers'
+import {BaseSizeScale} from '../constants'
```

Comment:
Should this include "condensed", "normal", and "spacious"? I think we include these in most of our spacing use cases.

Reply:
The base scale is a number array, which corresponds directly to our base spacing tokens (from 4 to 128). the terms like 'condensed', 'normal' and 'spacious' are semantic / functional names so they aren't related to our size scale. The reusing of those names is coincidental.